CHANGELOG.md

```diff
@@ -5,6 +5,11 @@
 ### Fixed
 
 -   DOM nesting console warnings in `<InfoListItem>`'s `title`, `subtitle` and `info` props ([#644](https://github.com/brightlayer-ui/react-component-library/issues/644)).
+-   Rendering of chevron and right component in `<InfoListItem>` at the same time ([#364](https://github.com/brightlayer-ui/react-component-library/issues/364)).
```

Comment:
I recommend updating **all** issue links in this changelog to point to their new location in the new etn-ccis org. It should be a quick find-and-replace.

```diff
- https://github.com/brightlayer-ui/react-component-library/issues/364
+ https://github.com/etn-ccis/blui-react-component-library/issues/364
```